src/views/studio/CreateMemberView/CreateMemberView.tsx

```diff
@@ -97,9 +134,11 @@ const CreateMemberView = () => {
           error={!!errors.about}
           helperText={errors.about?.message}
         />
-        <StyledButton type="submit">Create membership</StyledButton>
+        <StyledButton disabled={nodeConnectionStatus !== 'connected'} type="submit">
+          Create membership
+        </StyledButton>
       </Form>
-      <ActionDialog showDialog={loading} exitButton={false}>
+      <ActionDialog showDialog={isLoadingDialogVisible} exitButton={false}>
```

Comment:
Let's do this for errors: Let's skip displaying them in the form altogether. And we can probably remove the friendly messages as well. In theory, our UI logic should make sure none of the errors ever shows up because we do all the checks on our own. So if there's an error from the faucet let's just show an error dialog with something like this. `Some unexpected error occurred. Error code: HandleTooLong`